src/sonic-yang-models/yang-models/sonic-system-ldap.yang

```diff
@@ -0,0 +1,128 @@
+module sonic-system-ldap {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-system-ldap";
+    prefix ssys;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-port {
+        prefix port;
+    }
+
+    import sonic-portchannel {
+        prefix lag;
+    }
+
+    import sonic-loopback-interface {
+        prefix loopback;
+    }
+
+    import sonic-interface {
+        prefix interface;
+    }
+
+    import sonic-mgmt_port {
+        prefix mgmt-port;
+    }
+
+    description "LDAP YANG Module for SONiC OS";
+
+    revision 2023-10-01 {
+        description "First Revision";
+    }
+

```

Comment:
Do we intend to add filter options , which would be useful in optimizing queries
T<filterT> MAP FILTER
The FILTER is an LDAP search filter to use for a specific map. The default filter is a basic search on the objectClass for the map (e.g. T<(objectClass=posixAccount)T>).

Reply:
Adance LDAP configuration can be raised for future work.
The motivation is to have first a stable feature version.
And then we can discuss who can lead advances in LDAP configurations